src/patternfly/components/FileUpload/file-upload.scss

```diff
@@ -0,0 +1,132 @@
+  --pf-c-file-upload__text-area--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-file-upload__text-area--MinHeight: calc(var(--pf-global--spacer--3xl) * 2);
+  --pf-c-file-upload__text-area--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-file-upload__text-area--BorderTopColor: var(--pf-global--BorderColor--300);
+  --pf-c-file-upload__text-area--BorderRightColor: var(--pf-global--BorderColor--300);
+  --pf-c-file-upload__text-area--BorderBottomColor: var(--pf-global--BorderColor--200);
+  --pf-c-file-upload__text-area--BorderLeftColor: var(--pf-global--BorderColor--300);
+  --pf-c-file-upload__text-area--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-file-upload__text-area--hover--BorderBottomColor: var(--pf-global--active-color--100);
+  --pf-c-file-upload__text-area--active--BorderBottomColor: var(--pf-global--active-color--100);
+  --pf-c-file-upload__text-area--focus--BorderBottomColor: var(--pf-global--active-color--100);
+  --pf-c-file-upload__text-area--active--BorderBottomWidth: var(--pf-global--BorderWidth--md);
+  --pf-c-file-upload__text-area--focus--BorderBottomWidth: var(--pf-global--BorderWidth--md);
+  --pf-c-file-upload__text-area--readonly--BackgroundColor: var(--pf-global--disabled-color--300);
+
+  // Button overrides
+  --pf-c-file-upload--c-button--m-control--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-file-upload--c-button--m-control--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-file-upload--c-button--m-control--disabled--BackgroundColor: var(--pf-global--disabled-color--300);
+  --pf-c-file-upload--c-button--m-control--disabled--after--BorderTopColor: var(--pf-global--BorderColor--300);
+  --pf-c-file-upload--c-button--m-control--disabled--after--BorderRightColor: var(--pf-global--BorderColor--300);
+  --pf-c-file-upload--c-button--m-control--disabled--after--BorderBottomColor: var(--pf-global--BorderColor--200);
+  --pf-c-file-upload--c-button--m-control--disabled--after--BorderLeftColor: var(--pf-global--BorderColor--300);
+  --pf-c-file-upload--c-button--m-control--disabled--after--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-file-upload--c-button--m-control--OutlineOffset: calc(-1 * var(--pf-global--spacer--xs));
+
+  display: flex;
+  flex-direction: column;
+
+  &.pf-m-draganddrop {
+    position: relative;
+    border: var(--pf-c-file-upload--m-draganddrop--BorderWidth) solid var(--pf-c-file-upload--m-draganddrop--BorderColor);
+
+    &::after {
+      position: absolute;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      content: "";
+      background-color: var(--pf-c-file-upload--m-draganddrop--after--BackgroundColor);
+      opacity: var(--pf-c-file-upload--m-draganddrop--after--Opacity);
+    }
+  }
+}
+
+.pf-c-input-group > .pf-c-form-control {
+  &::placeholder {
+    color: var(--pf-c-file-upload--c-input-group--c-form-control--placeholder--Color);
+  }
+}
+
+.pf-c-file-upload__text-area {
```

Comment:
I would make this a `div` that you place a `textarea.pf-c-form-control` inside of